app/gui/src/components/TabView.vue

```diff
@@ -83,11 +91,19 @@ const onSignOut = () => {
           @update:selected="$event && setPage(project.id)"
         >
           <SvgIcon v-if="readyProjects.has(project.id)" name="graph_editor" />
-          <LoadingSpinner v-else :size="16" />
+          <GrowingSpinner v-else :state="loadingProjectSpinnerState(project)" :size="16" />
           <span>{{ projectNames.get(project.id) }}</span>
-          <SvgIcon name="close" @click="closeProject(project)" />
+          <SvgButton
+            class="closeButton"
+            :class="{ onMacOs: isOnMacOS() }"
```

Comment:
A `computed` would be slightly more efficient (due to less need for Vue prop diffing), and more obviously right (A function being run on every component update could be a performance bug; looking at the implementation, this function is probably ok to call frequently, but I'd prefer to be able to see that it's ok without needing to check the function implementation.)

Reply:
I had a different vision of how vue compute properties: I assumed they're not recomputed if there was no reactive dependency change. Here we have no reactive dependencies. I mean, if _I_ would be a vue developer, probably this is the way I would implement that :)

even if I'm wrong, a constant in setup should be as good as computed.